spec/models/reports/repository_spec.rb

```diff
@@ -223,7 +231,8 @@ def refresh_views
 
       facility_1_results = result[facility_1.slug]
       range.each do |period|
-        expect(facility_1_results[period]).to_not be_nil
+        # Not sure we really care about this behavior
```

Comment:
We should probably decide rather than comment it out. AFAICT this is just testing that there's some data for each period in the requested range. It's not the highest quality test, more of a canary in the coalmine than a proper unit test. Can it easily be made better by stating something like, `expect(facility_1_results).to match_array([...])`? Or is this behavior tested elsewhere?

Reply:
Agreed, I'll be revisiting this in more repository PRs I'm pretty sure.